apps/vendor-portal/src/modules/import/pages/import-new.vue

```diff
@@ -350,6 +344,13 @@ export default defineComponent({
       ];
     });
 
+    const sampleTemplateUrl = computed(() => {
+      return selectedImporter.value
+        ? process.env.PLATFORM_URL.slice(0, -1) +
```

Comment:
I am not sure that we have `process.env.PLATFORM_URL` on the environments. `slice` can fail due to `undefined` env.